build-system/tasks/css.js

```diff
@@ -86,10 +86,13 @@ function compileCss(watch, opt_compileAll) {
    */
   function writeCss(css, jsFilename, cssFilename, append) {
     return toPromise(
-      // cssText is hardcoded in AmpCodingConvention.java
-      file(jsFilename, 'export const cssText = ' + JSON.stringify(css), {
-        src: true,
-      })
+      file(
+        jsFilename,
+        '/** @noinline */ export const cssText = ' + JSON.stringify(css),
```

Comment:
Did you verity this works?

Reply:
Yes, it works. I was waiting for #23363 to be reviewed, after which I'll rebase this PR on `master` and post a full diff of `v0.js`.

Reply:
Done in https://github.com/ampproject/amphtml/pull/23373#issuecomment-512517487